libs/openFrameworks/utils/ofUtils.cpp

```diff
@@ -469,10 +454,7 @@ bool ofToBool(const string& boolString) {
 	if(lower == "false") {
 		return false;
 	}
-	bool x = false;
-	istringstream cur(lower);
-	cur >> x;
-	return x;
+	return (bool)std::stoi(boolString);
```

Comment:
sorry, i didn't test `ofToBool("False");`…
i will test tomorrow more cases.

but i think:
if `boolString` is `False`, then `lower = ofToLower(boolString)` is `false`.
and match the case of `lower == "false"`.
therefore, return `false`.